granulate_utils/linux/ns.py

```diff
@@ -295,6 +295,8 @@ def get_host_pid(nspid: int, container_id: str) -> Optional[int]:
         try:
             if container_id in Path(f"/proc/{process.pid}/cgroup").read_text():
                 pid_namespace = os.readlink(f"/proc/{process.pid}/ns/pid")
+                if os.readlink(f"/proc/{process.pid}/ns/pid") == pid_namespace and get_process_nspid(process) == nspid:
```

Comment:
`os.readlink(f"/proc/{process.pid}/ns/pid") == pid_namespace` how it can be different value?

Reply:
Woops